common/lifestyle_perks/wc_death_magic_perks.txt

```diff
@@ -0,0 +1,423 @@
+﻿death_magic_tree_1_perk_1 = {
+	lifestyle = death_magic_lifestyle
+	tree = death_magic_tree_1
+	position = { 2 0 }
+	icon = node_learning
+
+	auto_selection_weight = {
+		value = 11
+		if = {
+			limit = { has_trait = magic_good_noninheritable }
+			add = 1989
+		}
```

Comment:
I feel like you would like to create `script_value` here. It's very common.

Reply:
Yeah, I have no idea why, but paradox don't use `script_value` in lifestyles

Reply:
Done